### ISSUE TITLE
Data Aggregation

### ISSUE THREAD

**Issue by stdavis** [agrc/raster#171]
For the v3, we are planning on greatly reducing the number of feature classes involved with this app. The individual project tile index feature classes will be merged into six product feature classes. Rick has created a preliminary version of this data here: `<agrc share>\raster\Raster_3.0\Raster.gdb`.

**Comment by rkelson** [comment]
The six different feature classes are available from L:\agrc\raster\Raster_3.0\Raster.gdb

- Aerial_Photography
- AutoCorrelated_DEMs
- Contours
- DRGs
- Lidar
- USGS_DEMs

**Comment by stdavis** [comment]
This is waiting on a decision on where to store these new feature classes that will be accessible to the web app. The current thought is to put them into internal and let them get pushed out to AGOL and OpenSGID. Then the app can point at their corresponding AGOL services. There is some concern about making these data publicly available.

**Comment by stdavis** [comment]
> Could the layers not be made public and only shared with our organization?

No, those are mutually exclusive. Because raster.utah.gov does not have any authentication, we can only hit services that are public.
